824 display more information on the extractor popup

longshuicy opened the pull request:
<img width="1450" alt="image" src="https://github.com/clowder-framework/clowder2/assets/13950475/799afddc-2912-4a24-bfe6-03098b938782">
<img width="972" alt="image" src="https://github.com/clowder-framework/clowder2/assets/13950475/6dfa732e-e642-49e1-8235-e3277cd190bf">
<img width="965" alt="image" src="https://github.com/clowder-framework/clowder2/assets/13950475/8fde74ba-9ae8-43e6-8b4d-55a15046eb98">

lmarini reviewed (changes requested):
Looks great. Three minor suggestions:
- Could we get a close button or x in top right for popup?
- The tables on the files and the extraction history tabs have borders. To be consistent should we have one here as well?
- Not changed in this PR, but when an extractor is not alive, should we still be able to click on the title and get the description? Right now we can't.
Thanks!

longshuicy commented:
> Looks great. Three minor suggestions:
> 
> * Could we get a close button or x in top right for popup?
> * The tables on the files and the extraction history tabs have borders. To be consistent should we have one here as well?
> * Not changed in this PR, but when an extractor is not alive, should we still be able to click on the title and get the description? Right now we can't.
>   Thanks!

I think i addressed all of them. i also rewrote the searchbox (using generic search component we have), as well as align them better. 
<img width="1454" alt="image" src="https://github.com/clowder-framework/clowder2/assets/13950475/80a5ba1d-6b46-4781-8b3e-e27dfe345136">
<img width="930" alt="image" src="https://github.com/clowder-framework/clowder2/assets/13950475/a0436aaf-d030-4c62-9ffd-7c23c1fa2c93">

tcnichol reviewed (approved):
This looks good. I like the expanded info on both the extractor page and on submit. 

Marking approved.
